src/app/features/fss-search/fss-search.component.html

```diff
@@ -33,13 +33,13 @@ <h3>{{messageTitle}}</h3>
       
       <app-fss-simplified-filter id="gotoFilterMenu" *ngIf="activeSearchType === SearchTypeEnum.SimplifiedSearch" tabindex="-1"
       (onApplyFilterButtonClicked)="onApplyFilterButtonClicked($event)" [filterGroups]="filterGroups"></app-fss-simplified-filter>
-      <div >
-        <h2>
+      <div>
+        <h2 id="skipResultsLink" tabindex="-1">
           <span class="totalResultLabel" *ngIf="searchResultTotal == 1" role="alert">{{this.searchResultTotal}} result found</span>
           <span class="totalResultLabel" *ngIf="searchResultTotal > 1" role="alert">{{this.searchResultTotal}} results found</span>
           <span #showSearchResult *ngIf="searchResultTotal > pageRecordCount" class="paginatorLabel" role="alert"> {{this.paginatorLabel}}</span>
         </h2>
-        <app-fss-search-results class="SimplifiedSearchResult" id="skipResultsLink" tabindex="-1" [searchResult]="searchResult"
+        <app-fss-search-results class="SimplifiedSearchResult" [searchResult]="searchResult" [currentPage] = "currentPage"
```

Comment:
Please remove the spaces around the equals
```suggestion
        <app-fss-search-results class="SimplifiedSearchResult" [searchResult]="searchResult" [currentPage]="currentPage"
```

Reply:
updated as suggested. Thanks